virtio-devices/src/balloon.rs

```diff
@@ -138,6 +179,49 @@ impl BalloonEpollHandler {
     }
 
     fn release_memory_range(
+        &mut self,
+        memory: &GuestMemoryMmap,
+        range_base: GuestAddress,
+        range_len: usize,
```

Comment:
I think it's better to take `pfn` as input, from which we can calculate the range_base and range_len. This will ensure the context is clear that virtio-balloon only works for 4K requests.

Reply:
If use pfn instead of "base" and "len" we should convert it to "base" and "len" every iteration.

Reply:
For balloon inflation, the len will always be 4K, and `pfn` and`GuestAddress` are 1:1. So  you don't need iteration at all.

Reply:
"release_memory_range" is also used other place like "process_reporting_queue" where len is taken from desc. So, should we make it common?

Reply:
No, I think it is better to have separate function to handle different situations, so that each of them is cleaner.

Reply:
OK

Reply:
@jongwu I would suggest the following:
1. Keep the original `release_memory_range()` as it is. So that no changes are needed from `process_reporting_queue()`;
2. Create a new function for handling balloon inflation. You can simply combine ___your___ "fn release_memory_range()" into your `release_memory_range_4k`.

Reply:
Really, it's better.